src/data/fintype/basic.lean

```diff
@@ -992,11 +992,14 @@ have injective (e.symm ∘ f) ↔ surjective (e.symm ∘ f), from injective_iff_
 λ hsurj, by simpa [function.comp] using
   e.injective.comp (this.2 (e.symm.surjective.comp hsurj))⟩
 
+lemma card_of_bijective {f : α → β} (hf : bijective f) : card α = card β :=
+card_congr (equiv.of_bijective f hf)
```

Comment:
This proof seems a little silly because `card_congr` simply extracts the bijectivity proof and throws the equiv away! It's not worth pausing bors for, but ideally you'd prove `card_congr` in terms of this lemma.